controls/cis_ocp_1_4_0/section-5.yml

```diff
@@ -128,13 +128,14 @@ controls:
       level: level_2
   - id: '5.5'
     title: Extensible Admission Control
-    status: pending
+    status: partial
     rules: []
     controls:
     - id: 5.5.1
       title: Configure Image Provenance using image controller configuration parameters
-      status: pending
-      rules: []
+      status: partial
+      rules:
+        - general_configure_imagepolicywebhook
```

Comment:
This rule needs some work, so the partial status is correct. But, shouldn't we keep notes or links to cards in that case?
Additionally, I wonder if the rules `ocp_allowed_registries`, `ocp_allowed_registries_for_import`, `ocp_insecure_registries` could be used here?

Reply:
I think those rules are relevant, but they will fail by default, and we don't have a remediation for it, wondering if this might be too strict to add here

Reply:
I'm ok with the rules failing by default since that's inline with what's documented in the benchmark (we say that image provenance is not setup by default, and we provide links to users to help them configure image registries).

That approach here seems reasonable since it's highlighting a legitimate finding for users.